code-challenges.test.js

```diff
@@ -71,4 +121,20 @@ var vowelTester3 = "challenge"
 
 
 
-// b) Create the function that makes the test pass.
+// b) Create the function that makes the test pass
+//Create a function that takes a string and returns the first vowel
+const firstVowelIndex = (string) => {
+//initiate a variable that equals vowels both lowercase and uppercase
+    let vowelList = 'aeiouAEIOU'
+//initate another variable set to 0
+    let vowelIndex = 0
+//for the length of the string
+    for(var i= 0; i < string.length ; i++)
+    {
+//write a conditional that finds the index of the first vowel
+        if (vowelList.indexOf(string[i]) !== -1)
+    vowelIndex += 1
```

Comment:
Great Pseudo Coding, This function returns an output that isn't expected though. Your logic is headed in the right direction but I think your if() statement isn't evaluating what you want because it doesn't have a return.